usage_examples/multitask_openai/README.md

```diff
@@ -0,0 +1,46 @@
+# Performing multiple tasks in a single pipeline
+
+This example shows how you can perform multiple LLM-backed tasks within
+a single spaCy pipeline.
+
+We could create a new custom task that performs all objectives in a single
+LLM query, but in this example we'll only use built-in task templates to
+see how easy it is to compose them. Note that breaking down tasks this way
+might be a better choice anyway, since it allows you to better control the
+performance of your pipeline.
+See [Matt Honnibal's blog post on the subject](https://explosion.ai/blog/against-llm-maximalism)
+for more context.
```

Comment:
Ok, seeing it like this makes me think it's better to leave the link out after all. Like the rephrasing!
```suggestion
```